transformer_engine/pytorch/module.py

```diff
@@ -2147,7 +2147,7 @@ def forward(
                 activation_dtype,
                 get_workspace(),
                 bias=fc1_bias,
-                use_bias=True,
```

Comment:
As far as I can tell, this old behavior of always setting `bias=True` for FC1 in `LayerNormMLP` was a discrepancy from the Megatron reference. This PR seems to match the reference.